.github/workflows/tests.yml

```diff
@@ -61,8 +61,8 @@ jobs:
     - uses: pypa/cibuildwheel@v2.9.0
       env:
         CIBW_ARCHS_MACOS: universal2
-        CIBW_MANYLINUX_X86_64_IMAGE: manylinux1
-        CIBW_BUILD: cp39-win_amd64 cp36-manylinux_x86_64 cp38-macosx_universal2
+        CIBW_MANYLINUX_X86_64_IMAGE: manylinux2014
```

Comment:
This can go away now, it can (and is, I think) be set in pyproject.toml.

Reply:
Oh yes, I forgot that we use `pyproject.toml` as a configuration source even in the CI.